### ISSUE TITLE
[CrashlyticsPlugin] "Unstripped native library path does not exist" because check is done before the "assembleRelease" task creates it (with plugin version 2.9.3 and 2.9.4)

### ISSUE THREAD

**Issue by bowserf** [firebase/firebase-android-sdk#4682]
## Configuration
- Gradle version: `7.5.1`
- AGP: `7.2.2`

My application, using a native library, is configured to upload debug symbols on Firebase Crashlytics.
Here is its configuration:
```
android {
  buildTypes {
        release {
            configure<com.google.firebase.crashlytics.buildtools.gradle.CrashlyticsExtension> {
                nativeSymbolUploadEnabled = true
                strippedNativeLibsDir = "$buildDir/intermediates/stripped_native_libs/release/out/lib"
                unstrippedNativeLibsDir = "$buildDir/intermediates/merged_native_libs/release/out/lib"
            }
        }
    }
}
```

## Problem:
Starting from firebase-crashlytics-gradle version `2.9.3`, when I do an "assembleRelease" (because I configured the plugin in release) without a previous build cache, I got the following error after a few seconds:
> Unstripped native library path does not exist: /app/build/intermediates/merged_native_libs/release/out/lib. Either specify the correct unstrippedNativeLibsDir or disable Crashlytics symbol uploading.

It worked with previous plugin versions (we previously used version `2.9.2`.

It seems the existence of a directory at the targeted path is now done before the "assembleRelease" process and before creating such a directory.

I now have to:
- disable the plugin `nativeSymbolUploadEnabled = false`
- build my release
- re-enable the plugin `nativeSymbolUploadEnabled = true`
Before executing the Gradle task to upload native symbols.

**Comment by argzdev** [comment]
Thanks for reporting, @bowserf. I was able to reproduce the same behavior with version `2.9.4`. I'll notify our engineers and see what we can do here.

**Comment by soygabimoreno** [comment]
I had the same issue with `2.9.4`. It worked in `2.9.2`.

**Comment by Durkaen** [comment]
Hi, I am also having the same issue. Is there any news?

**Comment by bernhardpflug** [comment]
Same issue here! Worked fine with 2.9.2

**Comment by horne3754sg** [comment]
Hi, I just want to add my own findings. 

I have encountered this issue with 2.9.4 as well. It seems to me that it tries to include 'build/intermediates/merged_native_libs/release/out/lib' before its actually built them.

If I run 2.9.2, it will build the project as  thought I ran `npm run android`, but for release and generate a release variant of the relevant intermediate files. 

![image](https://user-images.githubusercontent.com/2529445/225541198-9ce3878c-6ffb-43c5-8e32-227da646ed9d.png)

But if I run 2.9.4 it fails and informs me that the file could not be found which makes sense because its not been built.

![image](https://user-images.githubusercontent.com/2529445/225541022-6945f4a8-fe3e-4bdf-9cf1-adc766046bb9.png)

**Comment by mrober** [comment]
Thanks for all the information. This issue happens when overriding the unstrippedNativeLibsDir. We have a fix for this that will go out in the next release, in about 2 weeks. For now, I recommend using version 2.9.2. I will update this after the release with the specific version.

**Comment by bowserf** [comment]
Thank you very much for your message and the fix that is coming 🙏 

We stay on `2.9.2` until you release the next version

**Comment by mrober** [comment]
This is fixed in `2.9.5` now. Please reopen this issue if you have any problems.

**Comment by soygabimoreno** [comment]
> This is fixed in `2.9.5` now. Please reopen this issue if you have any problems.

Thank you so much for fixing it, @mrober! 🙏
